site/content/config.js

```diff
@@ -12,6 +12,7 @@ export const config = {
   repoRoot: "https://github.com/flowershow/flowershow",
   repoEditPath: "/edit/main/site/content/",
   editLinkShow: false,
+  contentExclude: [],
```

Comment:
It would be good to also utilize the other contentlayer option: `contentDirInclude`, so that you can have all content excluded by default except for the small set of specific folders/files.

Reply:
You will also have to test what happens if you set both these values. I can't see any info on that in contentlayer's docs so I think it may work just fine, i.e. if you select a given subdirectory you want to publish and you want to exclude some child directory/file, e.g. include `content/docs` and exclude `content/docs/config.mdx`

Reply:
It works when including a folder and excluding a file in that folder